### PR TITLE
Update J-jobs and some scripts for matching NCO WCOSS2 requirements

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
Ming, I accidentally included the sms script updates in this PR.  I think you should be able to view the files that changed from the 2nd commit only though.

### CODE REVIEW

File: jobs/JHRRR_ANALYSIS [hunk 1]
```diff
@@ -103,24 +104,34 @@ setpdy.sh
...
-export DCOM=${DCOM:-${DCOMROOT}}
-export FVCOM=${FVCOM:-${DCOM}/${PDY}/wgrdbul/glerl}
-export FVCOMm1=${FVCOMm1:-${DCOM}/${PDYm1}/wgrdbul/glerl}
-
-export gespath=${gespath:-$GESROOT/${envir}}
-export HRRRGES_ENKF=${HRRRGES_ENKF:-$gespath/${NET}/hrrrges_enkf/${dom}}
-export HRRRGES_SFC=${HRRRGES_SFC:-$gespath/${NET}/hrrrges_sfc/${dom}}
-export HRRRDASGES=${HRRRDASGES:-$gespath/${NET}/hrrrdasges}
-export RAPGES_SATBIAS=${RAPGES_SATBIAS:-${GESROOT}/${envir}/rap/rapges_satbias}
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
```
Comment (hu5970) [inline jobs/JHRRR_ANALYSIS]: Do we need to define DCOMROOT in operation. Or it is an environmental variable. Steven has an email on DCOMROOT.

File: jobs/JHRRR_PREP_RADAR [hunk 2]
```diff
@@ -91,20 +91,27 @@ setpdy.sh
...
+export COMINradar=${COMINradar:-$(compath.py $envir/hourly/$radar_ver)}
 
 # Hourly Lightning Data
-#export COMINlight=${COMINlight:-$(compath.py rap/prod)}
-export COMINlight=${COMINlight:-${COMRAP}}
+export COMINlight=${COMINlight:-$(compath.py $envir/rap/$rap_ver)}
 
-# NSSL Mosaic Data
+# Hourly Mosaic Data
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
```
Comment (hu5970) [inline jobs/JHRRR_PREP_RADAR]: Double check this one.

File: jobs/JHRRR_PREP_REF2TTEN [hunk 3]
```diff
@@ -75,8 +76,7 @@ setpdy.sh
 ########################################
 # Define Input Data Directories
 ########################################
-# Hourly Radar Data
-export COM_RAD=${COM_RAD:-$(compath.py hourly/prod)}
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
```
Comment (hu5970) [inline jobs/JHRRR_PREP_REF2TTEN]: check this one

**Comment by BenjaminBlake-NOAA** [comment]
Based on Steven's email it looks like DCOMROOT is defined in operations by loading the prod_envir module.  Since we can't do that for testing, we had to hardwire the path.  But for the delivery I should comment out the DCOMROOT definitions in all the J-jobs.  I think I can do that on GitHub directly.

File: jobs/JHRRR_PREP_SMOKE [hunk 4]
```diff
@@ -80,18 +79,25 @@ setpdy.sh
...
 mkdir -m 775 -p $COMOUT 
 
 #######################################
 # Define observation source directories
 #######################################
-#export DCOM_MODIS="$DCOMROOT/prod/"
-#export DCOM_VIIRS_NPP="$DCOMROOT/prod/"
-#export DCOM_VIIRS_J01="$DCOMROOT/prod/"
+
+export DCOMROOT=${DCOMROOT:-/lfs/h1/ops/$envir/dcom}
```
Comment (hu5970) [inline jobs/JHRRR_PREP_SMOKE]: double check

**Review by hu5970 — APPROVED** [review]

**Pull request merged**